lib/bpk-component-text-link/src/BpkTextLink.ios.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import React from 'react';
+import { TouchableWithoutFeedback } from 'react-native';
+import { primaryColor } from 'bpk-tokens/tokens/base.react.native';
+
+import {
+  BpkDynamicStyleSheet,
+  useBpkDynamicStyleSheet,
+} from '../../bpk-appearance';
+import BpkText from '../../bpk-component-text';
+import { getThemeAttributes, withTheme } from '../../bpk-theming';
+
+import { type Props, propTypes, defaultProps } from './common-types';
+import { REQUIRED_THEME_ATTRIBUTES } from './theming';
+
+const dynamicStyles = BpkDynamicStyleSheet.create({
+  text: {
+    color: primaryColor,
+  },
+});
+
+const BpkTextLink = (props: Props) => {
+  const { children, onPress, accessibilityLabel, theme, ...rest } = props;
+
+  const styles = useBpkDynamicStyleSheet(dynamicStyles);
+  const themeAttributes = getThemeAttributes(REQUIRED_THEME_ATTRIBUTES, theme);
+
+  const themeStyle = {
+    text: themeAttributes && {
+      color: themeAttributes.textLinkColor,
+    },
+  };
+
+  return (
+    <TouchableWithoutFeedback
```

Comment:
Using this directly instead of `BpkTouchableOverlay` as it was causing the link not to render when passing children into it.

The Backpack component is just a wrapper around this component anyway so given the basic requirements we need I felt this could work as I spent 4days trying to work out why the Backpack component didn't work but couldn't see

Reply:
Seeing how this behave in practice I'm not sure this is the best approach, the react docs mention `Do not use unless you have a very good reason. All elements that respond to press should have a visual feedback when touched.`

Also, `BpkTouchableOverlay` as far as I remember is supposed to be used only on iOS, Android has `BpkTouchableNativeFeedback` for the same purpose.

Check the implementation of button for example, iOS is using BpkTouchableOverlay and Android BpkTouchableNativeFeedback, so I believe we should stick to the same approach.